src/content/docs/dropins/checkout/tutorials/add-payment-method.mdx

```diff
@@ -0,0 +1,140 @@
+
+<Tasks>
+
+<Task>
+### Prerequisites
+
+You must [install and configure](https://docs.adyen.com/plugins/adobe-commerce/set-up-the-plugin-in-adobe-commerce) the Adyen Payment Plugin on your Adobe Commerce backend before integrating it with the Commerce boilerplate template.
+
+<Aside type="note" title="Compatibility">
+The Adyen package is not compatible with [API Mesh](https://developer.adobe.com/graphql-mesh-gateway/).
+</Aside>
+
+</Task>
+
+<Task>
+
+### Add your client key
+
+After you install and configure the Adyen Payment Plugin on your Adobe Commerce backend, you need to add your Adyen client key to the Commerce boilerplate configuration.
+
+<Aside type="note">
+  You should use a development or staging environment for integration testing purposes before modifying your production configuration.
+</Aside>
+
+<Steps>
+1. Open your project's `configs-stage` or `configs-dev` Google Sheet or Microsoft Excel file.
+
+1. Add a new key-value pair for your Adyen API key.
+
+   | key               | value           |
+   |-------------------|-----------------|
+   | adyen-client-key  | YOUR_CLIENT_KEY |
+
+1. Save the file.
+1. Publish file changes with the [Sidekick](/get-started/#sidekick) extension.
+   
+</Steps>
+
+</Task>
+
+<Task>
+
+### Install the package
+
+<Steps>
+1. Open a terminal and navigate to your boilerplate-based project.
+1. Install the Adyen NPM package and add it to the list of project dependencies in your `package.json` file with the `--save` flag.
+   
+   ```bash
+   npm install @dropins/adyen-checkout-extension --save
+   ```
+</Steps>
+
+</Task>
+
+<Task>
+
+### Integrate with the boilerplate
+
+To integrate third-party payment providers, like Adyen, with Adobe Commerce, you can use the extensibility features provided by the checkout drop-in component. The checkout component allows you to customize the list of payment methods shown during the checkout process using slots.
+
+<Steps>
+1. Open the `blocks/commerce-checkout/commerce-checkout.js` file.
+
+1. Import the required files.
+   
+   ```js
+   ...
+   // Adyen drop-in modules
+   import * as adyenApi from '@dropins/adyen-checkout-extension/api.js';
+   import { render as AdyenProvider } from '@dropins/adyen-checkout-extension/render.js';
+   import AdyenPaymentMethod from '@dropins/adyen-checkout-extension/containers/AdyenPaymentMethod.js';
+   ...
+   ```
+
+1. Register the Adyen API.
+
+   ```js
+   ...
+   initializers.register(adyenApi.initialize, {});
+   ...
+   ```
+
+1. Use the `PaymentMethod` [handlers slot](/dropins/checkout/slots/) to add a payment method handler for Adyen. This handler will render the Adyen payment method UI when the backend indicates that Adyen credit card is an allowed payment method.
+
+   ```js
+   ...
+   context.addPaymentMethodHandler('adyen_cc', {
```

Comment:
It looks like `context.addPaymentMethodHandler` is replaced with something like:

```
render(PaymentMethodsContainer, {
  slots: {
    Handler: {
      adyen_cc: { render: .... }
```

Where the first argument to this function is now the object key and the second argument is the old object that has a `render` prop.